open-company/open-company-web
# Save local edit

Card: https://trello.com/c/nF4gLVHw

This uses [localforage](https://github.com/localForage/localForage) that has the benefits to create an abstraction layer between browsers. It supports  IndexedDB, WebSQL and localStorage.

To test:
- click new post button
- edit some data (headline, body, board or topic)
- click the closing X
- [x] click again New post, do you see the previous edits? Good
- make some more edits and save as draft
- go back to the previous board
- [x] click New post button, do you NOT see the previous edits? Good
- make some edits and dismiss
- navigate to another board
- [x] click New post, do you NOT see your last edits? Good (we don't want you to see a post started on another board)
- now publish a post
- open the published post
- enter edit mode and edit some data
- dismiss the edit mode
- [x] open again the previous post and enter edit mode, do you see your edits? Good
- make some more edits
- navigate away from the app (ie: click on a bookmark in your browser)
- [x] go back to edit that post, do you see you latest edits? Good
- now edit some more and the post your edits
- navigate away from our app
- [x] go back to edit, do you see the last published edits? Good
